opensrp-chw/src/main/java/org/smartregister/chw/activity/AncMemberMapActivity.java

```diff
@@ -0,0 +1,259 @@
+                double north = boundingBox.north();
+                double south = boundingBox.south();
+                double east = boundingBox.east();
+                double west = boundingBox.west();
+
+                if (userLocation.getLatitude() > north) {
+                    north = userLocation.getLatitude();
+                } else if (userLocation.getLatitude() < south) {
+                    south = userLocation.getLatitude();
+                }
+
+                if (userLocation.getLongitude() > east) {
+                    east = userLocation.getLongitude();
+                } else if (userLocation.getLongitude() < west) {
+                    west = userLocation.getLongitude();
+                }
+
+                mapboxMap.animateCamera(CameraUpdateFactory.newLatLngBounds(LatLngBounds.from(north, east, south, west), BOUNDING_BOX_PADDING));
+            }
+        }
+    }
+
+    @Nullable
+    private BoundingBox showCommunityTransporters(@NonNull MapboxMap mapboxMap, @Nullable FeatureCollection featureCollection) {
+        if (featureCollection != null && featureCollection.features() != null && featureCollection.features().size() > 0 && communityTransportersSource != null) {
+            //CameraPosition cameraPosition = new CameraPosition.Builder(). featureCollection.bbox();
+            BoundingBox boundingBox = featureCollection.bbox();
+
+            if (boundingBox == null) {
+                double[] bbox = TurfMeasurement.bbox(featureCollection);
+                boundingBox = BoundingBox.fromLngLats(bbox[0], bbox[1], bbox[2], bbox[3]);
+            }
+
+            mapboxMap.animateCamera(CameraUpdateFactory.newLatLngBounds(LatLngBounds.from(boundingBox.north(), boundingBox.east(), boundingBox.south(), boundingBox.west()), BOUNDING_BOX_PADDING));
+
+            communityTransportersSource.setGeoJson(featureCollection);
+            return boundingBox;
+        }
+
+        return null;
+    }
+
+    @Nullable
+    private FeatureCollection loadCommunityTransporters() {
+        return FeatureCollection.fromFeatures(getRespondersFeatureList());
+    }
+
+    private List<Feature> getRespondersFeatureList() {
+        List<CommunityResponderModel> communityResponders = ChwApplication.getInstance().communityResponderRepository().readAllResponders();
+        List<Feature> featureList = new ArrayList<>();
+        int counter = 0;
+        for (CommunityResponderModel communityResponderModel : communityResponders) {
+            try {
+                Feature feature = getFeature(communityResponderModel);
+                feature.addNumberProperty(RECYCLER_VIEW_POSITION_PROPERTY, counter);
+                featureList.add(feature);
+
+                counter++;
+            } catch (JSONException e) {
+                Timber.e(e);
+            }
+        }
+        return featureList;
+    }
+
+    private com.mapbox.geojson.Feature getFeature(CommunityResponderModel communityResponderModel) throws JSONException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'com.mapbox.geojson.Feature' due to existing import 'com.mapbox.geojson.Feature'](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5493232)